apps/src/applab/firebaseCounters.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
```

Comment:
`'use strict';` is no longer needed; babel-loader for webpack implicitly inserts it at the top of every module.